src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.h

```diff
@@ -31,11 +31,11 @@ namespace app {
 namespace Clusters {
 namespace MicrowaveOvenControl {
 
-constexpr uint32_t kDefaultCookTime    = 30u;
-constexpr uint8_t kDefaultPowerSetting = 100u;
-constexpr uint32_t kMaxCookTime        = 65535u;
-constexpr uint32_t kMinCookTime        = 1u;
-constexpr uint16_t kDerivedModeTag     = 16384u;
+constexpr uint32_t kDefaultCookTime = 30u;
+constexpr uint32_t kMinCookTime     = 1u;
+constexpr uint8_t kDefaultMinPower  = 10u;
+constexpr uint8_t kDefaultMaxPower  = 100u;
+constexpr uint8_t kDefaultPowerStep = 10u;
```

Comment:
Probably power should also have `Watt` as a suffix throughout.

Reply:
those 3 fields are used for power setting in number, I have changed all the '`powerSetting`' to '`powerSettingNum`' as well.